cmd/explaintest/config.toml

```diff
@@ -1,6 +1,7 @@
 port = 4001
 lease = "0"
 mem-quota-query = 34359738368
+apply-cache-quota = 104857600
```

Comment:
Should this config be put into the `performance` section?

Reply:
I think it's better to put `apply-cache-quota` and `mem-quota-query` together. So it doesn't put it into the `performance` section.